crates/txtx-addon-kit/src/types/types.rs

```diff
@@ -426,7 +427,18 @@ impl Value {
                 bytes
             }
             Value::Addon(data) => data.bytes.clone(),
-            _ => unimplemented!(),
+            Value::Integer(value) => value.to_be_bytes().to_vec(),
+            Value::Float(value) => value.to_be_bytes().to_vec(),
+            Value::Bool(value) => vec![*value as u8],
+            Value::Null => vec![],
+            Value::Object(values) => {
+                let mut joined = vec![];
+                for (key, value) in values.iter() {
+                    joined.extend(key.as_bytes());
+                    joined.extend(value.to_bytes());
+                }
+                joined
+            }
```

Comment:
Did you hit that codepath during your tests?
I also have some fixes in the solana branch iirc.